boinc-client/global_prefs_override.xml

```diff
@@ -1,6 +1,6 @@
 <global_preferences>
    <max_ncpus_pct>100.000000</max_ncpus_pct>
-   <ram_max_used_busy_pct>100.000000</ram_max_used_busy_pct>
-   <ram_max_used_idle_pct>100.000000</ram_max_used_idle_pct>
+   <ram_max_used_busy_pct>95.000000</ram_max_used_busy_pct>
```

Comment:
Curious about this, what difference this 5% make?

Reply:
@tmigone on a 4GB device, it would mean 200MB ram is reserved, which should hopefully enable the supervisor, UI and the rest of the OS to continue to function normally. It's less on 2GB and 1GB of course, but hopefully the small reduction as well as the reduction in core usage will work there.